packages/create-sitecore-jss/src/initializers/vue/args.ts

```diff
@@ -0,0 +1,4 @@
+﻿import { ClientAppArgs } from './../../common/args/base';
+import { VueAnswer } from './prompts';
+
+export interface VueArgs extends ClientAppArgs, Partial<VueAnswer> {}
```

Comment:
Also extend `StyleguideArgs`
```suggestion
export interface VueArgs extends ClientAppArgs, StyleguideArgs, Partial<VueAnswer> {}
```